doc/backing-store-crd.md

```diff
@@ -17,11 +18,23 @@ Adding support for a new type of backing-store is rather easy as it requires jus
 
 # Reconcile
 
-#### OBC type
+#### AWS-S3 type
```

Comment:
Please add a code example how to create each backing store type from the CLI and also the yaml.